spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/modal/ModalBottomSheet.kt

```diff
@@ -124,21 +157,23 @@ import kotlin.math.max
  * @param dragHandle Optional visual marker to swipe the bottom sheet.
  * @param content The content to be displayed inside the bottom sheet.
  */
+
 @Composable
 @ExperimentalMaterial3Api
-public fun ModalBottomSheet(
+internal fun SparkModalBottomSheet(
     onDismissRequest: () -> Unit,
     modifier: Modifier = Modifier,
+    showHandle: Boolean = true,
     sheetState: SheetState = rememberModalBottomSheetState(),
     shape: Shape = BottomSheetDefaults.ExpandedShape,
     containerColor: Color = BottomSheetDefaults.ContainerColor,
     contentColor: Color = contentColorFor(containerColor),
     tonalElevation: Dp = BottomSheetDefaults.Elevation,
     scrimColor: Color = BottomSheetDefaults.ScrimColor,
     dragHandle: @Composable (() -> Unit)? = {
-        BottomSheetDefaults.DragHandle(Modifier.padding(vertical = 8.dp))
+        if (showHandle) BottomSheetDefaults.DragHandle(Modifier.padding(vertical = 8.dp))
```

Comment:
Use null instead of Unit here since the api can be nullable otherwise it can't know that there is no content